src/include/psc.h

```diff
@@ -37,6 +37,7 @@ enum {
 enum {
   KIND_ELECTRON,
   KIND_ION,
+  KIND_ELECTRON_HE,
```

Comment:
Since this is a generic header, it's not really the right place to define the special HE electrons, and it shouldn't be needed outside of `psc_flatfoil_yz.cxx`)